File: Iniciante/1015/1015.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    
    public static double dist(Scanner teclado){
        double x1 = teclado.nextFloat();
        double y1 = teclado.nextFloat();
        double x2 = teclado.nextFloat();
        double y2 = teclado.nextFloat();
        return Math.sqrt(Math.pow(x1-x2,2) + Math.pow(y1-y2,2));
    }
 
    public static void main(String[] args) throws IOException {
        Scanner teclado = new Scanner(System.in);
        double distancia = dist(teclado);
        System.out.printf("%.4f\n", distancia);
    }
 
}

File: Iniciante/1015/1015.py
import math
X1,Y1 = map(float, input().split())
X2,Y2 = map(float, input().split())
calculo = (X2-X1)**2 + (Y2-Y1)**2
distanca_dois_pontos = math.sqrt(calculo)
print(f"{distanca_dois_pontos:.4f}")